src/OmniSharp/Models/HighlightResponse.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.CodeAnalysis.Classification;
+using Microsoft.CodeAnalysis.Text;
+
+namespace OmniSharp.Models
+{
+    public class HighlightResponse
+    {
+        public LinePosition Start { get; set; }
+        public LinePosition End { get; set; }
```

Comment:
I like using positions (Line,Column) but for the sake of consistency we should stick with ```StartLine```, ```StartColumn```, ```EndLine```, and ```EndColumn``` - it's used like that in other places.